### PR TITLE
MSC:1819 Remove Presence Lists

### PR DESCRIPTION
**Pull request by neilisfragile**
[Rendered](https://github.com/matrix-org/matrix-doc/blob/neilj/msc-remove-presence-lists/proposals/1819-remove-presence-lists.md)

### CODE REVIEW

**Comment by turt2live** [comment]
In the interest of getting this in for r0 (and because it looks great to me - thank you!):

@mscbot fcp merge

File: proposals/1819-remove-presence-lists.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Remove references to presence lists
+
+[Presence](https://matrix.org/docs/spec/client_server/r0.4.0.html#id388) lists allow a given user the ability to subscribe to other users and be alerted to their current presence status.
```
Comment (richvdh) [inline proposals/1819-remove-presence-lists.md]:
```suggestion
[Presence](https://matrix.org/docs/spec/client_server/r0.4.0.html#id107) lists allow a given user the ability to subscribe to other users and be alerted to their current presence status.
```

File: proposals/1819-remove-presence-lists.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+While spec'd, no established client has implemented support and the only server side implementation raises privacy concerns.
+
+The proposal is to simply remove references to presence lists with a view to revisiting the same ideas in the future.
+
+This MSC addresses [#1810](https://github.com/matrix-org/matrix-doc/issues/1810)
```
Comment (richvdh) [inline proposals/1819-remove-presence-lists.md]:
```suggestion
This MSC addresses [#1810](https://github.com/matrix-org/matrix-doc/issues/1810).
```

**Comment by turt2live** [comment]
A miscommunication and subsequent timing error on our part has lead to this MSC's FCP not scheduled to clear in time. Given the extensive opportunity for people to review, and our goal to cut a s2s r0 today, we are fast forwarding this FCP to end now. There have been no requested changes while this has been in FCP or before this proposal was even opened.

**Comment by turt2live** [comment]
This proposal has also jumped to merged because the spec PR has been reviewed and merged while this has been in FCP. No changes have been made to the MSC, and as such no further changes are required to the spec for this MSC.

**Pull request merged**